Created a new article: Problem with reinstalling AdGuard Ad Blocker o… (#983)
AdguardTeam/AdguardKnowledgeBase: 04.android/06.solving-problems/15.multiple-profiles-issue/docs.en.md

---

> title: 'Problem with reinstalling AdGuard on Android devices when using the app on more than one user account'

Reviewer: Better use "multiple users" term or "user profiles" instead of "account". It will sound less confusing this way.
Same for the Russian version.

Reviewer 2: I agree that we should follow the official Android documentation. [Here](https://source.android.com/devices/tech/admin/multi-user) they state that a profile is a subset, 'A profile is a subset of, and tied to, the existence of a user. A user can have multiple profiles. Profiles are created through a Device Administration application. A profile always has an immutable association to a parent user, defined by the user that created the profile. Profiles do not exist beyond the lifetime of the creating user.'

If it is possible to install/uninstall AdGuard for multiple user profiles, we should state that. 

Please explain why we should not use "user accounts" for Android? What exactly makes this definition confusing? It may be useful to refer to the same link above.

Reviewer: Maybe that's only my opinion, but it's not semantically clear what account is meant in this sentence (it can be any social account, such as Google). That's why I suggested to use that term.

taxonomy:
   category:
       - docs
visible: true
---
 

> In Android 9 and later, if AdGuard is installed for more than one user account on your device, you may encounter problems with removing the app. When you delete AdGuard from one user account, the app will still show up on the list of applications, but you can’t uninstall or reinstall it. It happens because AdGuard is installed for another user account on the device.

Reviewer 2: ```suggestion
In Android 9 and later, if AdGuard is installed for more than one user account on your device, you may encounter problems when uninstalling the app. When you uninstall AdGuard from one user account, the app will still appear on the app list, but you won't be able to uninstall or reinstall it. It happens because AdGuard is installed for a different user account on the device.
```

> If you try to reinstall AdGuard after a failed attempt of removal, you will see the error "You can't install the app on your device".

Reviewer 2: ```suggestion
If you try to reinstall AdGuard after an unsuccessful removal attempt, you will see the error message "You can't install the app on your device".
```

> To solve this problem you need to delete the application for all users: go to Settings -> All apps -> AdGuard. Tap three dots at the upper right corner and then tap *Uninstall for all users*.

Reviewer 2: ```suggestion
To solve this problem, you need to uninstall the application for all users: go to Settings -> All apps -> AdGuard. Tap three dots at the upper right corner and select *Uninstall for all users*.
```
